core/src/main/scala/filodb.core/memstore/MemStore.scala

```diff
@@ -142,7 +142,7 @@ trait MemStore extends ChunkSource {
    * in order of decreasing frequency/# of series per item.
    * @param topK the number of top items to return
    */
-  def indexValues(dataset: DatasetRef, shard: Int, indexName: String, topK: Int = 100): Seq[TermInfo]
+  def indexValues(dataset: DatasetRef, shard: Int, indexNames: String, topK: Int = 100): Seq[TermInfo]
```

Comment:
Lets rename this method to labelValues and the param indexNames to labelNames.
Keeps confusing me.